R/hugo.R

```diff
@@ -165,28 +166,32 @@ install_theme = function(
   force = FALSE, update_hugo = TRUE
 ) {
   r = '^([^/]+/[^/@]+)(@.+)?$'
-  r_zip = "\\.zip$"
-  theme_is_url = grepl(r_zip, theme)
+  theme_is_url = grepl('[.](zip|tar[.]gz)$', theme)
   if (!is.character(theme) || length(theme) != 1 || (!grepl(r, theme) & !theme_is_url)) {
-    warning("'theme' must be a character string of the form 'user/repo' or 'user/repo@branch', or a full URL to the .zip file")
+    warning(
+      "'theme' must be a character string of the form 'user/repo' or ",
+      "'user/repo@branch', or a full URL to the .zip or .tar.gz file"
+    )
     return(invisible())
   }
   branch = sub('^@', '', gsub(r, '\\2', theme))
   if (branch == '' || theme_is_url) branch = 'master'
   theme = gsub(r, '\\1', theme)
   dir_create('themes')
   in_dir('themes', {
-    if (theme_is_url) {
-      url = theme
-      zipfile = gsub(".+/(.+\\.zip)", "\\1", theme)
-    } else {
-      url = sprintf('https://%s/%s/archive/%s.zip', hostname, theme, branch)
-      zipfile = sprintf('%s.zip', basename(theme))
+    url = if (theme_is_url) theme else {
+      sprintf('https://%s/%s/archive/%s.tar.gz', hostname, theme, branch)
```

Comment:
@ijlyttle I changed the default archive format from zip to tar.gz. I guess it won't affect your use case (of GHE), but I just want to make sure. Thank you!

Reply:
Hi @yihui - sorry I have been "checked out" for a while - I will confirm!

Reply:
No worries! (The earliest email that I haven't replied in my inbox was from Nov 2018, if that makes you feel better :)